examples/fields.go

```diff
@@ -3,14 +3,13 @@ package main
 
 import (
 	"fmt"
-
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 )
 
 func main() {
-	groupService, err := service.NewNsGroupService("1.1.1.1", "xxx", "xxx", "v1", true)
+	groupService, err := service.NewNimbleGroupService(service.WithHost("1.1.1.1"), service.WithUser("xxx"), service.WithPassword("xxx"), service.WithWaitForAsyncJobs())
```

Comment:
Let's keep the lines shorter.

Reply:
Sorry.  I must have missed something.  Should this be:

`groupService, err := service.NewNimbleGroupService().WithHost("1.1.1.1").WithUser("xxx").WithPassword("xxx").WithWaitForAsyncJobs())`

Reply:
@rgcostea I'm not the best person to answer this but I think it's a Golang thing (https://www.sohamkamani.com/golang/options-pattern/)

Reply:
hmm but NewNsGroupService() is deprecated because it forces users to specify all 5 parameters. NewNimbleGroupService() allows them to specify only the parameters that they need (e.g. in this example, they don't have to specify the api version)

Reply:
I see.  Sorry for the confusion.  This seems like it's the best we can do with golang.  Thanks for the pointers.